components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/store/dao/AppNotificationTemplateDAO.java

```diff
@@ -223,6 +221,8 @@ public void updateNotificationTemplate(NotificationTemplate notificationTemplate
                     String.format("Error while updating %s template %s of type %s from application %s in %s tenant.",
                             channelName, locale, displayName, applicationUuid, tenantId);
             throw new NotificationTemplateManagerServerException(error, e);
+        } catch (IOException e) {
+            throw new NotificationTemplateManagerServerException("Error while closing content stream.", e);
```

Comment:
can this error also occur while processing the stream, not just while closing it?

Reply:
Fixed with: https://github.com/wso2-extensions/identity-event-handler-notification/pull/280/commits/4298c197eb3d442b51f6f8689d1b5a9c0cd5af10